pkg/kubernetes/utils_test.go

```diff
@@ -1,10 +1,188 @@
 package kubernetes
 
 import (
-	"fmt"
+	"os"
+  "fmt"
```

Comment:
I think you can use `gofmt` for your formatting! 🙇🏼‍♂️

Reply:
Sorry, I think it's because of i used github web IDE when i check conflict
fixed it. Thanks!